connectors/google/google-base/src/main/java/io/camunda/google/model/AuthenticationType.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ *             under one or more contributor license agreements. Licensed under a proprietary license.
+ *             See the License.txt file for more information. You may not use this file
+ *             except in compliance with the proprietary license.
+ */
+package io.camunda.google.model;
+
+import com.google.api.client.util.Value;
+import com.google.gson.annotations.SerializedName;
+
+public enum AuthenticationType {
+  @Value("bearer")
+  @SerializedName("bearer")
```

Comment:
I can't find a reason, but when using custom Gson factory instead of default, the parser can't parse the value without ```@SerializedName```. It just came to me now that I haven't tried without ```@Value``` annotation. I'll try it and based on the result remove it or close the conversation

Reply:
Without ```@Value``` default parser doesn't work. So, leave both

Reply:
Please take a look at other enums and try to find a root cause. I believe, we use only one of those.

Reply:
Created separate issue #606